patches/tModLoader/Terraria/WorldGen.cs.patch

```diff
@@ -225,6 +225,15 @@
  
  			for (int n = 0; n < Main.countsAsHostForGameplay.Length; n++) {
  				Main.countsAsHostForGameplay[n] = false;
+@@ -4085,7 +_,7 @@
+ 
+ 			Main.tile[x, num7].slope(0);
+ 			Main.tile[x, num7].halfBrick(halfBrick: false);
+-			PlaceTile(x, num10, 135, mute: false, forced: true, -1, 7);
++			PlaceTile(x, num10, 135, mute: true, forced: true, -1, 7); // tML: muted post-sound-overhaul to avoid a crash
+ 			for (int m = num8; m <= num10; m++) {
```

Comment:
What crash? It should be fixed instead. What are the reproduction steps?

Reply:
literally just generate a world normally. as it is on current preview, attemptin' to generate a world has a number of errors which stem from non-muted `PlaceTile` calls scattered all throughout `WorldGen.cs`

Reply:
fixed in a better fashion by makin' `PlaceTile` calls never play a sound mid-worldgen